specs/signature-envelope-cose.md

```diff
@@ -56,6 +65,21 @@ Example of the Notary Project signature payload:
 }
 ```
 
+Example of the Notary Project blob signature payload:
```

Comment:
Payload format is same for blob and OCI. IMO we dont need desperate example for blob and oci.